bindings/kotlin/build.gradle.kts

```diff
@@ -42,7 +46,6 @@ tasks.register<KtfmtFormatTask>("KtfmtFormat") {
 // Generic task to run any example
 tasks.register<JavaExec>("example") {
     classpath = sourceSets["main"].runtimeClasspath
-    jvmArgs = listOf("-Djna.library.path=${projectDir}/lib")
```

Comment:
So even this was not needed ?

Reply:
The sourceSet is needed to have the binaries included in the JAR, the jvmArgs was separated (to let it know where to look for the binaries) but is apparently not needed as this is the path where it looks for anyways it seems